src/main/java/nextstep/ladder/LadderController.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.ladder;
+
+import nextstep.ladder.entity.User;
+import nextstep.ladder.entity.ladder.*;
+import nextstep.ladder.view.LadderHeightInputView;
+import nextstep.ladder.view.LadderResultView;
+import nextstep.ladder.view.UsersInputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LadderController {
+
+    public void ladderGame() {
+        // 사용자 입력 받기
+        Users users = getUsers();
+        // 사다리 높이 입력 받기
+        LadderHeight ladderHeight = getLadderHeight();
+
+        // 사다리 생성
+        FloorGenerator floorGenerator
+                = new FloorGenerator(RandomLinkGenerator.getInstance(), new LadderLine(users.getUserCount()));
+        Ladder ladder = Ladder.create(ladderHeight, floorGenerator);
+
+        // 출력
+        LadderResultView.display(ladder, users);
+    }
+
+    private Users getUsers() {
+        List<User> users = UsersInputView.getUserInput()
+                .stream()
+                .map(User::new)
+                .collect(Collectors.toList());
+        return new Users(users);
```

Comment:
이 로직을 Users에서 수행하면 어떤 장점이 있을까요 :) ?
```suggestion
        return new Users(UsersInputView.getUserInput());
```

Reply:
제 생각은
생성 로직을 캡슐화하여(응집도 상승) 외부에서 생성로직에 대해 몰라도 된다..?
인데 맞는걸까요? 😯

Reply:
넵 맞습니다! 생성 책임을 Users 객체에게 위임함으로써 스스로 생성을 할 수 있게되고, 외부에서 생성 로직에 대해 몰라도 되는 장점이 있다고 생각합니다!